### ISSUE TITLE
ליצור דף באתר: גלריית יישומים

### ISSUE THREAD

**Issue by niryariv** [hasadna/datacity-k8s#186]
גריד ריבועים שלושה לרוחב

כל אחד מכיל:

כותרת
צילום מסך (או תמונה אחרת שנצרף)
תיאור קצר
לפרטים צרו קשר (מייל של דורית)

* [שער לעסקים](https://br7biz.org.il/)
<img width="400" alt="" src="https://user-images.githubusercontent.com/1771/107199487-2861b100-69ff-11eb-84c3-ba44ceb2e7bd.png">
צ׳טבוט שמסייע ליזמים שרוצים לפתוח עסק בהתנהלות מול גופי העיריה השונים. הושק בשיתוף עיריית באר שבע

* [מפת עסקים](https://geoapp.datacity.org.il/?c=ashdod_business)
<img width="400" src="https://user-images.githubusercontent.com/1771/107199868-9c9c5480-69ff-11eb-8f93-304e5aec6b75.png">
אפליקציית הדגמה - מפת העסקים בעיר אשדוד

* [מפת חינוך](https://geoapp.datacity.org.il/?c=alqasum)
<img width="400" alt="Screen Shot 2021-02-08 at 11 20 58" src="https://user-images.githubusercontent.com/1771/107199953-ba69b980-69ff-11eb-8078-4bfe6d8a0014.png">
אפליקציית הדגמה - מפת מוסדות החינוך במועצה איזורית אל קסום

כותרת משנה: בפיתוח. תאריך יעד- אביב 2021

* מפתח תקציב עירוני
<img width="328" alt="Screen Shot 2021-02-08 at 12 06 11" src="https://user-images.githubusercontent.com/1771/107205017-0c154280-6a06-11eb-819b-2dc2d5c5cd55.png">

* כלי תכנון אסטרטגי מבוסס נתוני נגישות אמת בתחבורה ציבורית

<img width="400" src="https://user-images.githubusercontent.com/1771/107205278-60b8bd80-6a06-11eb-8e38-d0d112e175bc.png">

![image]()

* מפה לי - הנגשת מידע גיאוגרםי בקליק
<img width="400" src="https://user-images.githubusercontent.com/1771/107205668-c9a03580-6a06-11eb-8142-bed32fe65e1f.png">

* מדד מגדר עירוני פתוח

**Comment by akariv** [comment]
@OriHoch 
I'm currently working on this, lmk what you think of this proposed approach:
- Add to the Hasadna CKAN instance a new datatype (via the scheming mechanism) for 'Application'
- Each application will have the required fields Nir described or more

Apps would be editable and viewable on opendata.hasadna.org.il (and later on possibly in municipal portals - in case they want it).
We could pull the list of apps via the API and show them in a website with specialised frontend (I assume that's the requirement here)

**Comment by OriHoch** [comment]
yes, makes sense, but if it's just meant for this one purpose, seems it will be easier to do it using the datacity wordpress

if you do want to do it this way, it should be done in https://github.com/hasadna/ckanext-datacity - where we already have some schemas we add for settings and automation

**Comment by OriHoch** [comment]
we also have the pages extension installed on all ckan instances, so that's also a good option to implement it per-site (although limited by markdown)

**Comment by akariv** [comment]
The plan is to later use this as a database of applications, so a more
data-oriented approach is needed here.

**Comment by niryariv** [comment]
created [this](https://www.datacity.org.il/applications/) unglamorous page as a placeholder for now
